FiducialRegistrationWizard/Logic/vtkPointMatcher.h

```diff
@@ -63,6 +66,17 @@ class VTK_SLICER_FIDUCIALREGISTRATIONWIZARD_MODULE_LOGIC_EXPORT vtkPointMatcher
     // The mean distance error between each pair of points, after matching
     double ComputedRootMeanSquareDistanceErrorMm;
 
+    // Determine if there is more than one plausible mapping.
```

Comment:
Not clear what mapping you mean.

Reply:
Would it be clearer if I said point-to-point mapping?

Reply:
Somewhat better. It is still not clear what point to what point. Something like "reordering of from/to markup points for best match of positions" would be more clear.